lti/lti-common/src/java/org/sakaiproject/lti13/LineItemUtil.java

```diff
@@ -324,7 +324,10 @@ public static Map<String, String> getExternalIdsForToolAssignments(String contex
 				retval.put(assignmentReference, constructExternalId(content, null));
 			}
 		} catch (Throwable e) {
```

Comment:
Generally you shouldn't catch Throwable
```suggestion
		} catch (Exception e) {
```